src/style.css

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+body {
+  margin: 0;
+}
+
+.block {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block-snake {
+  height: 300px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex: 1 1 33.33%;
+}
+
+.block-snake:first-child {
+  background-color: #f00;
+}
+
+.block-snake:nth-child(2) {
+  background-color: #c00;
+}
+
+.block-snake:nth-child(3) {
+  background-color: #900;
+}
+
+.block-snake:nth-child(4) {
+  background-color: #600;
+  order: 2;
+}
+
+.block-snake:nth-child(5) {
+  background-color: #300;
+  order: 1;
+}
+
+.block-snake:nth-child(6) {
+  background-color: #000;
+}
+
+@media (max-width: 751px) {
```

Comment:
Still breakpoints are not correct.
Here's the hint from the task:
![image](https://user-images.githubusercontent.com/60885495/125265626-afe48080-e30d-11eb-81eb-a7d1dc385d11.png)

At 755px you have:
![image](https://user-images.githubusercontent.com/60885495/125265546-9e02dd80-e30d-11eb-95cc-4d317ea8d7dc.png)